packages/loader/container-loader/src/containerContext.ts

```diff
@@ -49,66 +48,84 @@ type ContextLifecycleEvents = "runtimeInstantiated" | "disposed";
 
 export class ContainerContext implements IContainerContext {
 	public static async createOrLoad(
-		container: Container,
+		options: ILoaderOptions,
 		scope: FluidObject,
```

Comment:
we could save come bits over the wire by making this a props object we pass down, fine to rest expand it end if we need to, but then there would only be 2 giant lists, create, and expand, rather than 4.

Reply:
I think some of this will go away entirely or at least change with the followup I mention - I'll take a look there first.